tasks/task843_financial_phrasebank_classification.json

```diff
@@ -0,0 +1,10888 @@
+{
+    "Contributors": [
+        "Jalansh Munshi"
+    ],
+    "Categories": [
+        "Classification"
+    ],
+    "Source": [
+        "(financial_phrasebank) https://huggingface.co/datasets/financial_phrasebank"
+    ],
+    "Definition": "In this task, you're given a piece of financial news in natural language. Your task is to classify the news based on its content into three classes: positive, negative, and neutral. Output must be 'positive', 'negative', or 'neutral'.",
```

Comment:
Looks good! 

It might be helpful to densify the definitions a bit:

```
In this task, you're given a piece of financial news in natural language. Your task is to classify the news based on its content into three classes: positive, negative, and neutral. Output must be 'positive', 'negative', or 'neutral'.
```

can be shortened to: 
```
Classify the given a piece of financial news into three classes: positive, negative, and neutral. Output must be 'positive', 'negative', or 'neutral'.
```

Reply:
You can apply the same idea for the rest of the tasks.

Reply:
@danyaljj I have made the changes. Can you please review it once and mark this comment as resolved if there are no more suggestions?